File: URI/JavaScript/uri-1014.js
// URI - 1014: https://www.urionlinejudge.com.br/judge/pt/problems/view/1014

const readline = require('readline')
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const travelledDistance = parseInt(input)

    rl.on('line', input => {
        const spentFuel = parseFloat(input)

        console.log((travelledDistance / spentFuel).toFixed(3) + ' km/l')

        rl.close()
    })
})

File: URI/Ruby/uri-1014.rb
#URI - 1014: https://www.urionlinejudge.com.br/judge/pt/problems/view/1014

Total_distance_in_Km = gets.to_i
Total_fuel_spent = gets.to_f
Average_consumption = Total_distance_in_Km / Total_fuel_spent
puts "#{'%.3f' % Average_consumption} km/l"